src/buildSummary.ts

```diff
@@ -26,14 +18,29 @@ export function getSummaryRows(buildSummary: string): any[] {
         if (t.failed) {
             return [t.name, '🔴 Failed', t.description, t.duration.toString()];
         } else if (t.skipped) {
-            return [t.name, '🔵 Skipped', t.description, t.duration.toString()];
+            return [t.name, '🔵 Skipped' + ' (' + interpretSkipReason(t.skipReason) + ')', t.description, t.duration.toString()];
```

Comment:
Seems like duration doesn't need the `toString` because it is a string already.

Reply:
That's correct! Verified and updated